kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -0,0 +1,1250 @@
+                # ovs-vswitchd will attempt DPDK EAL init on startup and
+                # needs hugepages to be available.
+                self._ensure_hugepages(
+                    dpdk_config.hugePageNumber, dpdk_config.hugePageSize)
+
+                r, o, e = bash.bash_roe('systemctl restart ovsdb-server')
+                if r != 0:
+                    raise Exception('failed to restart ovsdb-server: %s' % e)
+
+                self._clean_stale_vnics()
+
+                r, o, e = bash.bash_roe('systemctl restart openvswitch;'
```

Ruff (0.15.6)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)